tst/EnergyPlus/unit/Furnaces.unit.cc

```diff
@@ -1192,6 +1204,15 @@ TEST_F(EnergyPlusFixture, UnitaryHeatPumpAirToAir_MaxSuppAirTempTest)
     EXPECT_FALSE(state->dataFurnaces->CoolingLoad);
     // check if the air-to-air heat pump outlet temperature is capped at 45.0C
     EXPECT_NEAR(45.0, state->dataLoopNodes->Node(state->dataFurnaces->Furnace(1).FurnaceOutletNodeNum).Temp, 0.000001);
+    EXPECT_NEAR(0.3326, state->dataLoopNodes->Node(state->dataFurnaces->Furnace(1).FurnaceOutletNodeNum).MassFlowRate, 0.0001);
```

Comment:
This new test section actually passes without any code changes, because it's not a VS furnace. But leaving it in anyway.

Reply:
I assume this is the fan/furnace design flow rate. In the future (no change requested) I would rather see something that would change with future updates. For example` EXPECT_NEAR(state->dataFurnaces->Furnace(1).DesignMassFlowRate, ,,,` (or DesignMassFlowRate * PLR for cycling fan). Not sure if that works here or not but these high precision targets can easily change with various fixes and then the unit tests needs updating.